apps/fpc/fsw/src/fpc_msg.h

```diff
@@ -138,6 +198,42 @@ typedef struct
     uint8  ucCmdHeader[CFE_SB_CMD_HDR_SIZE];
 } FPC_NoArgCmd_t;
 
+
+/**
+**  \brief Add Message Flow command.
+*/
+typedef struct
+{
+    /** \brief cFE SB Cmd Msg Hdr */
+    uint8  ucCmdHeader[CFE_SB_CMD_HDR_SIZE];
+
+    /** \brief The param to update */
+    float  param;
+} FPC_UpdateParamFloatCmd_t;
+/**
+**  \brief Add Message Flow command.
+*/
+typedef struct
+{
+    /** \brief cFE SB Cmd Msg Hdr */
+    uint8  ucCmdHeader[CFE_SB_CMD_HDR_SIZE];
+
+    /** \brief The param to update */
+    int32  param;
+} FPC_UpdateInt32Cmd_t;
+/**
+**  \brief Add Message Flow command.
+*/
+typedef struct
+{
+    /** \brief TC message headers */
+    uint8          ucCmdHeader[CFE_SB_CMD_HDR_SIZE];
+
+    /** \brief Index of the channel for which this command applies */
+    boolean        param;
```

Comment:
Use "osalbool".  Unfortunately, there is no platform independent boolean data type.  Use osalbool so we can ensure that everybody uses the same base type.

Reply:
#185